algorithmia/manifest.json

```
@@ -0,0 +1,27 @@
+{
+  "display_name": "Algorithmia",
+  "maintainer": "support@algorithmia.io",
+  "manifest_version": "1.0.0",
+  "name": "algorithmia",
+  "metric_prefix": "algorithmia.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Monitor metrics for machine learning models in production",
+  "guid": "bb9defff-03ae-4a22-93a7-6db498d37cd7",
+  "support": "contrib",
+  "supported_os": [
+    "linux"
+  ],
+  "public_title": "Datadog-Algorithmia Integration",
+  "categories": ["monitoring"],
+  "type": "check",
+  "is_public": false,
+  "integration_id": "algorithmia",
+  "assets": {
+    "dashboards": {},
+    "saved_views": {},
+    "monitors": {},
+    "service_checks": "assets/service_checks.json",
+    "logs": {}
```

```suggestion
    "logs": {},
    "metrics_metadata": "metadata.csv"
```
This is a new field we've just introduced

---

Added, thanks!